app/views/content/train-to-be-a-teacher/get-school-experience.md

```diff
@@ -18,8 +19,8 @@ calls_to_action:
         link_target: "https://schoolexperience.education.gov.uk/"
 promo_content:
     - content/train-to-be-a-teacher/promos/mailing-list-promo
-navigation: 20.25
-navigation_description: Spending some time in a school can help you find out more about teaching and even help you decide what kind of teacher you want to be.
+navigation: 20.30
+navigation_description: Spending some time in a school can help you understand if teaching is right for you and give you experience to draw on in your application.
```

Comment:
@EllieNodder I think it is worth checking this wording with Myles - do we want to specifically be saying this here?

If we do, I think it probably needs rewording a bit as 'draw on' is a little fuzzy?

Reply:
@gemmadallmandfe I think Myles is on AL until 5th July - anyone else we could run this wording by?

Reply:
think the wording is better now - let's leave as is @EllieNodder but can you message Myles and double-check with him on this return?

Reply:
yep, will do 👍🏻